guides/v2.1/frontend-dev-guide/themes/theme-create.md

```diff
@@ -190,6 +190,18 @@ It is likely that your theme will also contain module-specific files, which are
 {:.bs-callout .bs-callout-info}
 When you change files stored here during theme development, you need to clear the `pub/static` and `var/view_preprocessed` directories, and then reload the pages. Otherwise, the old versions of files are displayed on the storefront.
 
+Run the command below to clear the `pub/static` directory:
+
+```bash
+rm -rf pub/static/*
```

Comment:
Please, specify the area as well. I believe it's better to say

```bash
rm -rf pub/static/frontend/*
```
Otherwise, there's a chance to kill pub/static/.htaccess file. And in 99% cases we don't need to remove generated assets for admin upon theme development.

Reply:
Thank you for notice, @rogyar !

Command `rm -rf pub/static/*` does not remove `.htaccess` in the `pub/static` directory. So, probably the best here is change text in the `{:.bs-callout .bs-callout-info}` section?

What do you think? 

Thank you!

Reply:
If your bash profile is configured for removing hidden files on using the wildcard, then troubles arise :) 
But it's rather an edge-case than some common issue. 
Ok, let's leave it as it is to keep things simple.